experimental/benchmarks/node-dc-eis/run-dc-eis.sh

```diff
@@ -16,9 +16,9 @@ MONGODB_PORT=27017
 export DB_URL="mongodb://127.0.0.1:${MONGODB_PORT}/node-dc-eis-db"
 
 #these may need changing when we find out more about the machine we're running on
-NODE_AFFINITY="numactl --physcpubind=0,3"
-MONGO_AFFINITY="numactl --physcpubind=1,2"
-RUNSPEC_AFFINITY="numactl --physcpubind=1,2"
+NODE_AFFINITY="numactl --physcpubind=0,4"
```

Comment:
why this change in the commit?

Reply:
I agree I don't think this is correct since I believe there are only 4 CPUs.